### PR TITLE
Add default value for dockerVersion 

### PR DESCRIPTION
**Pull request by inv-jishnu**
Add default value for dockerVersion parameter in build.gradle files to fix verification test failing in kelpie test build phase.

### CODE REVIEW

File: scalardb-test/build.gradle [hunk 1]
```diff
@@ -10,6 +10,7 @@ repositories {
 }
 
 def awssdkVersion = "2.14.24"
+def dockerVersion = "0.1.0"
```
Comment (miseyu) [inline scalardb-test/build.gradle]:
```suggestion
def dockerVersion = project.hasProperty('dockerVersion') ? project.dockerVersion : 'latest'
```

File: scalardl-test/build.gradle [hunk 2]
```diff
@@ -8,6 +8,8 @@ repositories {
     mavenCentral()
 }
 
+def dockerVersion = "0.1.0"
```
Comment (miseyu) [inline scalardl-test/build.gradle]:
```suggestion
def dockerVersion = project.hasProperty('dockerVersion') ? project.dockerVersion : 'latest'
```

File: scalardb-test/build.gradle (hunk 1 above)
Comment (inv-jishnu) [reply scalardb-test/build.gradle]: Thank you, I will change it.

File: scalardl-test/build.gradle (hunk 2 above)
Comment (inv-jishnu) [reply scalardl-test/build.gradle]: Thank you, I will change it.

**Review by miseyu — APPROVED** [review]
LGTM

**Review by feeblefakie — APPROVED** [review]
The changes look good, but what makes the verification fail?

Ah, OK, I got it. The new build.gradle uses the `dockerVersion`.

**Pull request merged**